gr-uhd/apps/uhd_app.py

```diff
@@ -179,8 +186,20 @@ def setup_usrp(self, ctor, args, cpu_format='fc32'):
                 ))
         self.antenna = self.usrp.get_antenna(0)
         # Set receive daughterboard gain:
-        self.set_gain(args.gain)
-        self.gain_range = self.usrp.get_gain_range(0)
+        if args.power:
+            if args.gain is not None:
+                print("[ERROR] Cannot provide both --gain and --power!")
```

Comment:
Prefer "Cannot parse" or something like that. The user can _provide_ both, but the app here can't use them both ...